packages/lit-html/src/lit-html.ts

```diff
@@ -557,9 +559,12 @@ const getTemplateHtml = (
   // malformed template in dev mode.
   // Returned as an array for terseness
   return [
-    // We don't technically need to close the SVG tag since the parser
-    // will handle it for us, but the SSR parser doesn't like that
-    html + strings[l] + (type === SVG_RESULT ? '</svg>' : ''),
+    // We don't technically need to close the SVG tag since the parser will
+    // handle it for us, but the SSR parser doesn't like that.
+    // Note that the html must end with a node after the final expression to
+    // ensure the last NodePart has an end node, hence adding a comment if the
+    // last string was empty.
+    html + (strings[l] || '<?>') + (type === SVG_RESULT ? '</svg>' : ''),
```

Comment:
Use `nodeMarker` instead of `'<?>'` to save a couple of bytes

Reply:
Heh that's what I started with, but `nodeMarker` will be interpreted as a `NodePart`, which is not what we want! We just need a static node, but not one with meaning.